src/components/CandidateCategoryTile.jsx

```diff
@@ -38,6 +38,7 @@ const CandidateCategoryTile = ({ slugs, category }) => {
                     <AppIcon
                       alt={t('app.logo.alt', { name: category })}
                       app={slug}
+                      type="konnector"
                       className="item-grid-icon"
```

Comment:
We should go deeper: Since we're in the candidate category it means that the konnector is not installed yet, right? If so, we can pass `priority={'registry'}` to the AppIcon in order to fetch the Icon directly from the registry instead of having a 404 from the stack and the querying the registry. No? @doubleface